.github/workflows/systest.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           filters: |
             nondoc:
               - '!**/*.md'
+              - '!.github/CODEOWNERS'
```

Comment:
This will not work as expected. Take a look at `ci.yml`. Without `predicate-quantifier: every` any point matching would set `nondoc` to true.